coops-ui/src/views/Dashboard.vue

```diff
@@ -236,6 +250,20 @@ export default {
           this.hasBlockerFiling = true
         }
       })
+    },
+    /**
+     * Toggle the Change of address warning dialog.
+     */
+    toggleCoaWarning () {
+      this.coaWarning = !this.coaWarning
```

Comment:
Aren't you always setting coaWarning to false here? I'd prefer more explicit code (and function names), just to keep it simple to understand.
(not a mandatory change)

Reply:
It is a toggle, so it will change it to the opposing state of whatever it currently is, to save having two methods ( one to close, one to open) or one method with conditionals.  I will certainly update the comments and method name!

Reply:
I get it, but we never using this function to display the warning, so it seems like overkill. Not a big deal though.

Reply:
Sorry, you were right. I wasn't calling the method to display the dialog, just to close it. So i adjusted it and now correctly am calling the toggle method to display it and calling the toggle method to close it. Thanks!

Reply:
IMO in this case I think it would be more clear to SHOW the dialog explicitly and then HIDE the dialog explicitly, instead of toggling it. The problem is you could get out of sync with the toggling, and then your call to show it might instead hide it and vice-versa. (JS errors cause interesting behaviours sometimes.)

However, let's leave it for now.

Reply:
Totally fair point, i understand where you're coming from completely. Being explicit leaves less room for any weird interactions that may occur from errors. I may have overcomplicated something simple like this, i think in my head i always try to process solutions that are dynamic in nature. Appreciate your feedback on this!